app/gui2/src/components/visualizations/GeoMapVisualization.vue

```diff
@@ -404,18 +404,26 @@ function pushPoints(newPoints: Location[]) {
       <button class="image-button"><SvgIcon name="geo_map_distance" /></button>
       <button class="image-button"><SvgIcon name="geo_map_pin" /></button>
     </template>
-    <div ref="mapNode" class="GeoMapVisualization" @wheel.stop></div>
+    <div ref="mapNode" class="GeoMapVisualization" @pointerdown.stop @wheel.stop></div>
   </VisualizationContainer>
 </template>
 
 <style scoped>
 .GeoMapVisualization {
   height: 100%;
 }
-</style>
 
-<style>
-.GeoMapVisualization > .mapboxgl-map {
-  border-radius: 16px;
+:deep(.mapboxgl-map) {
+  border-radius: var(--radius-default);
+}
+
+:deep(.mapboxgl-ctrl-attrib.mapboxgl-compact) {
+  min-height: 24px;
+}
+
+/* Copied from mapbox-gl CSS. */
+:deep(.mapboxgl-ctrl-attrib-button.mapboxgl-ctrl-attrib-button) {
+  background-color: hsla(0, 0%, 100%, 0.5);
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg width='24' height='24' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg' fill-rule='evenodd'%3E%3Cpath d='M4 10a6 6 0 1012 0 6 6 0 10-12 0m5-3a1 1 0 102 0 1 1 0 10-2 0m0 3a1 1 0 112 0v3a1 1 0 11-2 0'/%3E%3C/svg%3E");
```

Comment:
Why is it needed? We link mapbox-gl CSS in index.html. Add a broader explanation, please.

Reply:
added explanation.